client/index.html

```diff
@@ -7,7 +7,35 @@
     <title>Vite App</title>
   </head>
   <body>
-    <div id="app"></div>
+    <div id="app">
+      <header>
+        <h2>TO-DO APP WITH SERVER</h2>
+      </header>
+      <section id="Task-list">
```

Comment:
`Task-list` is not a good way to name
Use `task-list` or `tasklist` or `Task-List`
Always be consistent with your naming conventions
For JS: use `variableName` or `VariableName` or `variable_name`
For HTML: best to use `tag-name` or `tagname`